apps/common/src/main/scala/org/lfdecentralizedtrust/splice/store/HistoryMetrics.scala

```diff
@@ -164,4 +164,13 @@ class HistoryMetrics(metricsFactory: LabeledMetricsFactory)(implicit
     )(metricsContext)
 
   }
+
+  override def close(): Unit = {
+    UpdateHistoryBackfilling.completed.close()
```

Comment:
How does this interact with lazy vals? If the metric was not used until now, it will be created here and immediately closed? What's the benefit of using `lazy` then?

Reply:
Doesn't matter that it will be created then, that should be safe as long as we close it. We just don't want to report it if we don't use it all.